### ISSUE TITLE
PrivacyInfo.xcprivacy

### ISSUE THREAD

**Issue by xygkevin** [SVProgressHUD/SVProgressHUD#1120]
文件引用应该通过下面这种方式：
core.resource_bundles = {'SVProgressHUD' => ['SVProgressHUD/PrivacyInfo.xcprivacy']}

**Comment by honkmaster** [comment]
Please use English. Can you give any references why your solution is „right“?

**Comment by xygkevin** [comment]
when I add a PrivacyInfo.xcprivacy file to my project， I get an error
` ` `
Multiple commands produce  '/User/Xcode/DerivedData/Demo-dqdyrfxccschdiakhuirdvqwblex/Build/Products/Debug-iphoneos/Demo.app/PrivacyInfo.xcprivacy'

That command depends on command in Target 'Demo' (project 'Demo'): script phase “[CP] Copy Pods Resources”
` ` `

It seems that the library's PrivacyInfo.xcprivacy file is copied to the root directory of the project,  and then the conflict happened.

I referenced SDWebImage and verified that it works

Of course, there may be other ways to solve the error;

**Comment by A-Dirt** [comment]
> Please use English. Can you give any references why your solution is „right“?
If use core. resources, during project build, PrivacyInfo. xcprivacy will be copied to the root directory of IPA payload contents. If there is already a PrivacyInfo. xcprivacy in the root directory of IPA payload content, the project build will report an error; 
If you use core. resource_bundles={'SVProgressHUD '=>['SVProgressHUD/PrivacyInfo. xcprivacy']}, after `pod install`, PrivacyInfo. xcprivacy will be copied to SVProgressHUD. bundle  @honkmaster

**Comment by jiasongs** [comment]
> Please use English. Can you give any references why your solution is „right“?

`core.resources = ['SVProgressHUD/PrivacyInfo.xcprivacy']`, After unpacking the App：
<img width="238" alt="图片" src="https://github.com/SVProgressHUD/SVProgressHUD/assets/22904757/e5ab81ca-ae3d-44c2-9548-679be3aa8344">
Found that `PrivacyInfo.xcprivacy` is in the project root directory.

`core.resource_bundles = { 'SVProgressHUD_Privacy' => ['SVProgressHUD/PrivacyInfo.xcprivacy'] }`, After unpacking the App：
<img width="388" alt="图片" src="https://github.com/SVProgressHUD/SVProgressHUD/assets/22904757/56b57ebc-6a19-4d74-8baf-e95eb2efff2a">
Found that `SVProgressHUD_Privacy.bundle` is in the project root directory.

To sum up, if there is also `PrivacyInfo.xcprivacy` in the main project or other SDKs also have `core.resources = ['AnySDK/PrivacyInfo.xcprivacy']`, then `PrivacyInfo.xcprivacy` will be duplicated

**Comment by wolfcon** [comment]
This issue only exists when using `Static Framework` in CocoaPods.

**Comment by LittleWangYe** [comment]
In fact, if you don't set it like this: core.resource_bundles = {'SVProgressHUD' => ['SVProgressHUD/PrivacyInfo.xcprivacy']}. Then the privacy report file generated by Xcode will not contain SVProgressHUD's PrivacyInfo. To generate a privacy report, please refer to the following link: https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_data_use_in_privacy_manifests?language=objc.
